test/runtests.jl

```diff
@@ -8,5 +8,10 @@ include("append.jl")
 include("groups.jl")
 include("consistency.jl")
 include("loadsave.jl")
-include("recon.jl")
+
+# Only run the reconstruction tests on versions where `workspace` is a thing
+if VERSION < v"0.7.0-DEV.2917"
+    include("recon.jl")
```

Comment:
At some point we should run these on 0.7. The right approach is probably either to come up with some trickery involving replacing the module that contains the type so that they'll have to be reconstructed, or run the code to write the types in a separate process. But this can wait for another PR.